### PR TITLE
[ES]Translation of 04, 04-1

### PR DESCRIPTION
**Pull request by manuelpinar**
Hi Alf! I send you a pull request with the week 4 traslated. If you need something more tell me.

Regards

Manuel

### CODE REVIEW

File: docs/es/week04/04.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: Semana 4
+translation-date: 26 Mar 2020
+translator: Manuel Pinar-Molina
+---
+
+
+## Práctica
+
+Comenzamos con un breve resumen de álgebra lineal y luego extendemos el tema a convoluciones usando datos de audio como ejemplo. Se reiteran conceptos clave como localidad, estacionalidad y matriz de Toeplitz. Luego haremos una demostración del rendimiento de la convolución en el análisis de tono. Finalmente, hay una breve digresión sobre la dimensionalidad de los diferentes datos.
+
```
Comment (Atcold) [inline docs/es/week04/04.md]: Can you add the two parts for the lectures?

**Comment by Atcold** [comment]
Also, se #158, and add the English back as comment.
In this way we can trace what version has been translated.

**Comment by manuelpinar** [comment]
Hi, do it! I think that I share you correctly.

**Comment by manuelpinar** [comment]
Good morning Alf! I just modify the files 04-1 and 04 adding the comments of the text in english.

I hope It's right

File: docs/es/week04/04.md (hunk 1 above)
Comment (Atcold) [reply docs/es/week04/04.md]: The summaries for the two lectures are missing.
Also, you need to update the `_config.yml` file.

**Comment by Atcold** [comment]
2 things:

 1. Add the translation of the other two summaries.
 2. Edit the `_config.yml` file to add this contribution.

**Comment by manuelpinar** [comment]
Good morning Alf! Sorry for my inexperienced, but I don`t Know how I can share with you my changes!!! Yesterday I modified the 2 points that you ask me. Adding the translation and editing _config.yml. Do I need to make another pull request???

Regards 

Manuel
[week04 and _config.zip](https://github.com/Atcold/pytorch-Deep-Learning/files/4396733/week04.and._config.zip)

File: docs/_config.yml [hunk 2]
```diff
@@ -123,3 +123,9 @@ es:
     - path: es/week01/01.md
       sections:
         - path: es/week01/01-1.md
+    - path: es/week04/04.md
+      sections:
+        - path: es/week04/04-1.md
+    - path: es/week05/05.md
+      sections:
+        - path: es/week05/05-1.md
```
Comment (Atcold) [inline docs/_config.yml]:
```suggestion
```

**Comment by Atcold** [comment]
Let me try to be more clear.
The file `04.md` is only partially translated. Please, translate the whole file.
There is Lecture part A, Lecture part B, Practicum.
The first two translations are still missing.

**Comment by manuelpinar** [comment]
Hi alfred!
The week04 only have practicum part!! There isn't lectures in this week, neither in english version...

**Comment by Atcold** [comment]
Jajajajajajajajajajajajajajajajajajajaja!
Vale.

**Comment by Atcold** [comment]
Have to fix shit now 😭
Screenshot 2020-03-28 15 52 40

**Comment by Atcold** [comment]
Rebased on top of `master` and squash-merged.
Thanks!

**Comment by manuelpinar** [comment]
You are welcome!!!

**Comment by Atcold** [comment]
Creo que te olvidé de traducir los subtítulos.
Screenshot 2020-03-28 16 08 03

**Comment by Atcold** [comment]
Please, send a new PR with these additional translations.

**Comment by manuelpinar** [comment]
Ok, I'm sorry, I do it

**Comment by Atcold** [comment]
No need to be sorry 😛

**Pull request merged**